packages/flutter/test/widgets/basic_test.dart

```diff
@@ -380,6 +380,25 @@ void main() {
       expect(attributedHint.attributes[0].range, const TextRange(start: 1, end: 2));
     });
 
+    testWidgets('Semantics can set controls visibility of nodes', (WidgetTester tester) async {
```

Comment:
Consider adding a test that checks that identifiers from multiple nodes are aggregated when nodes are absorbed.